mucommander-protocol-adb/src/main/java/com/mucommander/commons/file/protocol/adb/AndroidMenu.java

```diff
@@ -98,7 +98,7 @@ public void menuSelected(MenuEvent e) {
         // Remove previous menu items (if any)
         removeAll();
 
-        List<String> androidDevices = AdbUtils.getDevices();
+        List<String> androidDevices = null; //AdbUtils.getDevices(); - defer loading like for Bonjour
```

Comment:
@ahadas defer it?

Reply:
sure, it doesn't worth the time on startup